server/config.coffee

```diff
@@ -34,7 +34,7 @@ config =
     ]
 
     plugins: [
-        MODEL_MODULE
+        'cozydb'
```

Comment:
cozydb handle it itself : if there is a db or dbName in americano options, it switch to pouchdb.
This should cover the cozy-light case. For the standole version to work, we need to set db or dbName from environement variable just above (https://github.com/aenario/cozy-emails/blob/d7efc8779304fb1abeb29d50b0bb9d2661018917/server.coffee#L15)